scenarios/evaluate/simulate_index/simulate_input_index.ipynb

```diff
@@ -0,0 +1,376 @@
+    "                \"content\": query,\n",
+    "            }\n",
+    "        ],\n",
+    "        max_tokens=800,\n",
+    "        temperature=0.7,\n",
+    "        top_p=0.95,\n",
+    "        frequency_penalty=0,\n",
+    "        presence_penalty=0,\n",
+    "        stop=None,\n",
+    "        stream=False,\n",
+    "    )\n",
+    "    message = completion.to_dict()[\"choices\"][0][\"message\"]\n",
+    "    # change this to return the response from your application\n",
+    "    return message[\"content\"]\n",
+    "\n",
+    "\n",
+    "async def callback(\n",
+    "    messages: List[Dict],\n",
+    "    stream: bool = False,\n",
+    "    session_state: Any = None,  # noqa: ANN401\n",
+    "    context: Optional[Dict[str, Any]] = None,\n",
+    ") -> dict:\n",
+    "    messages_list = messages[\"messages\"]\n",
+    "    # get last message\n",
+    "    latest_message = messages_list[-1]\n",
+    "    query = latest_message[\"content\"]\n",
+    "    context = None\n",
+    "    # call your endpoint or ai application here\n",
+    "    response = call_to_your_ai_application(query)\n",
+    "    # we are formatting the response to follow the openAI chat protocol format\n",
+    "    formatted_response = {\n",
+    "        \"content\": response,\n",
+    "        \"role\": \"assistant\",\n",
+    "        \"context\": {\n",
+    "            \"citations\": None,\n",
+    "        },\n",
+    "    }\n",
+    "    messages[\"messages\"].append(formatted_response)\n",
+    "    return {\"messages\": messages[\"messages\"], \"stream\": stream, \"session_state\": session_state, \"context\": context}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Generate Query Responses from index\n",
+    "In this example we use a search index as raw text generate Query Response pairs. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import requests\n",
+    "\n",
+    "\n",
+    "def generate_text_from_index(search_term: str) -> str:\n",
+    "    url = f\"{search_endpoint}/indexes/{index_name}/docs/search?api-version=2024-07-01\"\n",
+    "    headers = {\"api-key\": search_api_key, \"Content-Type\": \"application/json\"}\n",
+    "    search_query = {\"search\": search_term, \"top\": 10}\n",
+    "    response = requests.post(url=url, headers=headers, data=json.dumps(search_query))\n",
+    "\n",
+    "    text = \"\"\n",
+    "    if response.status_code == 200:\n",
+    "        results = response.json()\n",
+    "        for result in results[\"value\"]:\n",
+    "            text += result[\"description\"]\n",
+    "\n",
+    "    return text[:5000]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "real_estate_search_term = \"New York\"\n",
+    "text = generate_text_from_index(real_estate_search_term)"
```

Comment:
Add some more information about these:
- what does your index have (data/format of data)
- sample output of what a search on `New York` returns.
- Add some info to the "before you begin" section to link to a page where you can learn to create an index